frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -45,7 +45,10 @@ class ListDropdown_ extends React.Component {
       return;
     }
 
-    this.setState(({ selectedKey }) => {
+    this.setState(() => {
```

Comment:
We no longer need to wrap this in a `setState()` call since we're not using any previous props.